### PR TITLE
Add Zig support for the HTTP challenge

### PR DESCRIPTION
**Pull request by glokta1**

### CODE REVIEW

File: compiled_starters/zig/build.zig [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    // This declares intent for the executable to be installed into the
+    // standard location when the user invokes the "install" step (the default
+    // step when running `zig build`).
+    b.installArtifact(exe);
+
+    // This *creates* a Run step in the build graph, to be executed when another
+    // step is evaluated that depends on it. The next line below will establish
+    // such a dependency.
+    const run_cmd = b.addRunArtifact(exe);
```
Comment (rohitpaulk) [inline compiled_starters/zig/build.zig:20]: @glokta1 I think there are some changes required to make this accept args (which will be required in the stages related to files). Can you check and see what we're doing in other challenges?

File: starter_templates/zig/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+![progress-banner](https://codecrafters.io/images/default_progress_banners/http-server.png)
+
+This is a starting point for Go solutions to the
```
Comment (rohitpaulk) [inline starter_templates/zig/README.md:3]: I don't think you need a separate `README.md` file - `starter_templates/README.md` should work, no?

File: starter_templates/zig/codecrafters.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+#
+# These can be VERY verbose, so we suggest turning them off
+# unless you really need them.
+debug: false
+
+# Use this to change the Zig version used to run your code
+# on Codecrafters.
+#
+# Available versions: zig-0.12
+language_pack: zig-0.12
```
Comment (rohitpaulk) [inline starter_templates/zig/codecrafters.yml:11]: @glokta1 look at how other languages do this - They don't define separate `codecrafters.yml` files

**Review by rohitpaulk — CHANGES_REQUESTED** [review]
Notes added

**Review by rohitpaulk — APPROVED** [review]

**Comment by ryan-gang** [comment]
@glokta1 `test(zig)` tests didn't trigger for this PR. 
And I feel there is some issues there. All the currently open PRs are having `test(zig)` failing. 
Can you take a look at this please ?

**Comment by glokta1** [comment]
@ryan-gang Just merged a change that should fix it https://github.com/codecrafters-io/build-your-own-http-server/pull/70.

**Comment by ryan-gang** [comment]
Thanks @glokta1!

**Pull request merged**